model/src/main/scala/hmda/model/fi/lar/LoanApplicationRegister.scala

```diff
@@ -29,9 +29,65 @@ case class LoanApplicationRegister(
       s"|${denial.reason3}|$rateSpread|$hoepaStatus|$lienStatus"
   }
 
+  /**
+   * NOTE:  The DAT file format is not supported by CFPB
+   */
   def toDAT: String = {
-    //TODO: implement DAT output
-    ""
+    s"$id" +
+      padRight(respondentId, 10) + 
```

Comment:
So, the issue with both this field and the income field is that the generated LARs are never zero-filled.  So converting the generated LAR to a DAT output will never equal the original LAR (a generated income of 456 will be converted to 0456 in the DAT output.  I suppose modifying the tests would be the best way to address this problem, but it will unfortunately make the tests more complicated than a blanket "converted LARs must equal generated LARs".  Another possible solution would be to make sure that we always store both respondent ID and income with leading zeroes.